### DISCUSSION TITLE
SSL Certificates for pulling submodules for building under Windows Subsystem for Linux

### DISCUSSION THREAD

**Discussion by caysira** [probonopd/MiniDexed#269] [General]
I had problems to pull the git submodules for building under WSL on Windows 10. Always got ssl certificate errors:

fatal: unable to access 'https://codeberg.org/dcoredump/Synth_Dexed.git/': server certificate verification failed. CAfile: /etc/ssl/certs/ca-certificates.crt CRLfile: none

I googled a bit and found a solution on https://stackoverflow.com/a/63299750

I adapted the commands to get the ssl-certificates for codeberg.org:
```bash
openssl s_client -showcerts -servername codeberg.com -connect codeberg.com:443 < /dev/null 2>/dev/null | sed -n -e '/BEGIN\ CERTIFICATE/,/END\ CERTIFICATE/ p'  > codeberg-com.pem
cat codeberg-org.pem | sudo tee -a /etc/ssl/certs/ca-certificates.crt
```
Then I could pull all the submodules required to build MiniDexed.

I thought, I share it here to save noobs like me a few minutes.

Cheers,
Christian

**Comment by probonopd** [comment]
Thanks @caysira. Windows must have come a long way if it can build software like this these days :+1:
